src/config/constants/farmAuctions.ts

```diff
@@ -1,11 +1,10 @@
 import { Token as SDKToken, Pair, ChainId } from '@pancakeswap/sdk'
 import tokens from './tokens'
-import { FarmAuctionBidderConfig, Token } from './types'
+import { FarmAuctionBidderConfig } from './types'
 
-const getLpAddress = (token: string, quoteToken: Token) => {
+const getLpAddress = (token: string, quoteToken: SDKToken) => {
```

Comment:
This can now be renamed from SDKToken to just Token

Reply:
Done 👍